examples/plot_clip.py

```diff
@@ -0,0 +1,125 @@
+#    The object to be clipped will be clipped to the full extent of the clip
+#    object. If there are multiple polygons in clip object, data from shp will
+#    be clipped to the total boundary of all polygons in clip object.
+
+###############################################################################
+# Import Packages
+# ---------------
+#
+# To begin, import the needed packages. You will primarily use EarthPy's clip
+# utility alongside GeoPandas.
+
+import os
+import matplotlib.pyplot as plt
+import geopandas as gpd
+import earthpy as et
+import earthpy.clip as ec
+
+###############################################################################
+# Import Example Data
+# -------------------
+#
+# Once the packages have been imported, download the data needed for this
+# example: one line shapefile containing all major roads in North America,
+# and one polygon shapefile containing the United States boundary.
+
+data = et.data.get_data("spatial-vector-lidar")
+
+###############################################################################
+# Open Files with GeoPandas and Reproject the Data
+# -------------------------------------------------
+#
+# Start by setting your working directory. Then, import the data files to
+# GeoDataFrames using GeoPandas.
+#
+# Recall that the data must be in the same CRS in order to use the
+# ``clip_shp()`` function. If the data are not in the same CRS, be sure to use
+# the ``to_crs()`` function from GeoPandas to match the projects between the
+# two objects, as shown below.
+
+# Set your home environment
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+
+# Open both files with GeoPandas
+roads = gpd.read_file(
+    os.path.join(
+        "data",
+        "spatial-vector-lidar",
+        "global",
+        "ne_10m_roads",
+        "ne_10m_n_america_roads.shp",
+    )
+)
+country_boundary = gpd.read_file(
+    os.path.join(
+        "data", "spatial-vector-lidar", "usa", "usa-boundary-dissolved.shp"
+    )
+)
+
+# Reproject the roads layer to match the US boundary CRS
+roads = roads.to_crs(country_boundary.crs)
+
+# Plot the unclipped data
+# The plot below shows the roads data before it has been clipped
+fig, ax = plt.subplots(figsize=(12, 8))
+roads.plot(ax=ax)
+
+# Use the .boundary option in GeoPandas plotting to just plot the boundary of a polygon, instead of
+# a filled in version of the polygon. This allows for other data, such as the roads data, to be seen
+# in reference to the polygon boundary.
+country_boundary.boundary.plot(ax=ax, color="red")
+ax.set_title("Major NA Roads Unclipped to US Border", fontsize=20)
+ax.set_axis_off()
+plt.show()
+
+###############################################################################
+# Clip the Data
+# --------------
+#
+# Now that the data are opened as GeoDataFrame objects and in the same
+# projection, the data can be clipped! Recall that in this example, the roads
+# will be clipped to the United States boundary. To clip the data, make
```

Comment:
Last minor change - split the paragraph into two for improved readability, beginning with "To clip the data,..."